contracts/CG721.sol

```diff
@@ -1,40 +1,66 @@
-// SPDX-License-Identifier: MIT
+/// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.15;
 
+
+/** @dev Imports the OpenZeppelin implemtations of ERC721, Enumerable extension for 
+  * inherited functions like totalSuppy(), Counters to automically generate the next token id after every mint, 
+  * and ownable
+  * for the onlyOwner modifer
+  */
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
+/** @title CG 721
+  */  
 contract CG721 is ERC721Enumerable, Ownable {
   using Strings for uint256;
   using Counters for Counters.Counter;
   Counters.Counter private tokenIds;
-  uint256 public maxSupply = 20;
+  uint256 public maxSupply = 200;
   string public baseExtension = ".json";
-  string baseURI = '';
+  string baseURI = 'https://gateway.pinata.cloud/ipfs/QmTd3uKQp42cQ4dt2ztJNJVgJSnjQ36iYzKzwhuj9S1vD2/';
 
     constructor() ERC721("CG NFTs", "CG-NFT") {
     }
 
-    // Only owner to set IPFS base
-    function setBaseURI(string memory _newBaseURI) public onlyOwner {
+    /** @dev Sets the base URI for the generated Token URI
+      * @param _newBaseURI URL of IPFS folder gateway of JSON Files
+      */
+    function setBaseURI(string memory _newBaseURI) external onlyOwner {
```

Comment:
if U don’t plan to update it later or hack :) , I’d suggest to use it as a constructor parameter. Although, this is for internal usage, let’s keep our code looking good just in case our company’s repo will be reviewed by some entity or other devs.

Reply:
👍🏻 :) Thanks!